askbot/conf/static_settings.py

```diff
@@ -28,9 +31,10 @@ class AskbotStaticSettings(AppConf):
         }
 
     QUESTION_PAGE_BASE_URL = pgettext('urls', 'question') + '/'
+    print QUESTION_PAGE_BASE_URL
     SERVICE_URL_PREFIX = 's/' # prefix for non-UI urls
     SELF_TEST = True # if true - run startup self-test
-    TRANSLATE_URL = True # set true to localize urls
+    TRANSLATE_URL = False # set true to localize urls
```

Comment:
is this variable used? 
`TRANSLATE_URL = settings.ASKBOT_TRANSLATE_UR`
is better?